Add SiteGround elePHPant configuration and photos

ymihaylov opened the pull request:
Hello! We in SiteGround, created a new elePHPant who we give as a gift in Raffles, Quizzes, Events etc. I would like to add him to your website if you agree. :)

Here's how he looks in the list:

![Screenshot 2020-12-01 at 0 28 35](https://user-images.githubusercontent.com/1996948/100673963-10b9dc80-336d-11eb-839d-46bbc26a8adc.png)

Thank you!

cookieguru commented:
A description would be good: note the unique features of this elePHPant and explain how it can be obtained

ymihaylov commented:
Description added :)

ymihaylov commented:
@cookieguru  Can I ask you for a review of the description, and please correct me if I should change something! Thank you!

cookieguru commented:
Sounds like a lot of marketing; this site is just a reference of the different variants

ymihaylov commented:
Okay, I will think about it. Thank you, again!

ymihaylov commented:
Update description.

cookieguru commented:
Hello Groundy!

ymihaylov commented:
Updated elephpant's name to Groundy corresponding [this tweet](https://twitter.com/SiteGround/status/1339859180895789058/photo/1).

Happy New Year! :)

ymihaylov commented:
Hello! Should I add something else to this request?

philipsharp commented on source/_elephpants/2020-11-26-siteground-elephpant.md:
```diff
@@ -0,0 +1,19 @@
+tags:
+    - black
+name: Groundy
+sponsor: SiteGround Web Hosting Company
```
"Web Hosting Company" should be removed as it's not a part of the company name.

philipsharp commented on app/config/sculpin_site.yml:
```diff
@@ -85,6 +85,9 @@ subspecies:
         latin: "arcus"
+    siteground:
+        common: "SiteGround"
+        latin: "SiteGround elephantus"
```
This doesn't work as a subspecies name. Typically we use the name or something related, loosely translated into Latin. Since "SinceGround" isn't a real word, it's a little harder. Is there a good name we can translate?

philipsharp commented:
@ymihaylov Sorry for the delay. PRs on this project don't always get reviewed quickly.

ymihaylov commented:
@philipsharp Okay, I changed the names, rebased my branch to the latest changes in your repo, and squashed the commits. Can you check the request again?
